backend/controllers/users.js

```diff
@@ -23,46 +23,11 @@ module.exports = userController = {
   },
   getAll: async (req, res) => {
     try {
-      const users = await userManager.getAllusers();
```

Comment:
this needs to be changed too
.getAllVolunteers()

Reply:
i dont know why but this part i have same on mine comp))))